### PR TITLE
Brand-Consolidation - Burials Hub - populate link values

### PR DESCRIPTION
**Pull request by ncksllvn**
This PR fills in the links on the Burial Hub, and moves the page from `/burials/` to `/burials-memorials/` to be consistent with the rest of the website.

- Also contains work in this PR - department-of-veterans-affairs/vets-website/pull/8357
- Also fixes the layout for `/education/check-post-9-11-gi-bill-benefit-status/`

Related ticket - department-of-veterans-affairs/vets.gov-team/issues/12774

### CODE REVIEW

**Comment by mnorthuis** [comment]
A couple of tweaks

- [ ] Request Military Records - This item should not link anywhere yet
- [ ] VA Survivor Rates - This link needs the "www" in front of it (users on VA network cannot access without it)
- [ ] Parents Dependency...Rates - Needs "www"
- [ ] Beneficiary Financial Counseling - Needs "www"
- [ ] Fudiciary Services - Needs - "www"

**Comment by DanielleSoCompany** [comment]
Couple of things:

- [ ] A link within a page is going to the Survivors and Dependents page that we removed. I've updated it in this PR and requested a review from @bethpotts (always want a final editorial look on things before merging in case of typos/etc). Beth, can you please review and then request the merge from @ncksllvn? Thank you!
- [ ] Not sure what's going on on [this page](https://vetsgov-pr-8355.herokuapp.com/burials-memorials/veterans-burial-allowance/) or if it has anything to do with this specific work, but I noticed that instead of the "How do I apply?" copy and button, it's showing this error message:
image

**Comment by ncksllvn** [comment]
> A link within a page is going to the Survivors and Dependents page that we removed. I've updated it in this PR and requested a review from @bethpotts (always want a final editorial look on things before merging in case of typos/etc). Beth, can you please review and then request the merge from @ncksllvn? Thank you!

I merged those changes into this PR since they're related.

**Comment by ncksllvn** [comment]
> Not sure what's going on on this page or if it has anything to do with this specific work, but I noticed that instead of the "How do I apply?" copy and button, it's showing this error message:

That's powered by React, and is complicated because the React application code needs to be shared by Vets.gov. I've captured the work in this ticket - department-of-veterans-affairs/vets.gov-team/issues/12939 - so that we can go ahead and get this work merged in.

**Comment by DanielleSoCompany** [comment]
Hi @ncksllvn. Reviewed the rest of the pages. Saw a few other things. Thanks!

**Related to URLS**
- [ ] This url (https://vetsgov-pr-8355.herokuapp.com/burials-memorials/what-to-expect-at-military-funeral/) is showing a blank page. Other pages that link to this page are linking to the old url (https://vetsgov-pr-8355.herokuapp.com/burials-and-memorials/what-to-expect-at-a-funeral/).

**Other**
- [ ] A few of the links/descriptions on the hub page seem like they don't have enough space around them
- [ ] The accordions don't seem to be opening on this page: https://vetsgov-pr-8355.herokuapp.com/burials-memorials/dependency-indemnity-compensation/

File: va-gov/pages/burials-memorials/index.md [hunk 1]
```diff
@@ -1,55 +1,188 @@
...
+  <div class="link">
+    <a href="https://benefits.va.gov/Compensation/current_rates_dic.asp"><b>VA Survivor Benefits Rates (VA DIC)</b></a>
+    <p>View current dependency and indemnity compensation rates for surviving spouses and children.</p>
+
+  <div class="link">
+    <a href="https://benefits.va.gov/Pension/current_rates_Parents_DIC_pen.asp"><b>Parents Dependency and Indemnity Compensation (DIC) Rates</b></a>
+    <p>View current survivor benefit rates for parents.</p>
+
+  <div class="link">
+    <a href="https://www.archives.gov/veterans/replace-medals.html"><b>Replace Medals, Awards, and Decorations</b></a>
```
Comment (RobAWilkinson) [inline va-gov/pages/burials-memorials/index.md]: This is dead link, I think supposed to be: www.cem.va.gov/recmed_records.asp

Comment (RobAWilkinson) [reply va-gov/pages/burials-memorials/index.md]: nvm, that leads right back to that dead link 👎

**Review by RobAWilkinson — APPROVED** [review]

**Pull request merged**